build/generate_bundle.sh

```diff
@@ -33,6 +33,15 @@ generate_bundle() {
     ${OPERATOR_SDK} generate bundle --channels ${BUNDLE_CHANNELS} --default-channel ${BUNDLE_DEFAULT_CHANNEL} --manifests --metadata --version "${OPERATOR_BUNDLE_VERSION}" --output-dir "${WORKING_DIR}" >> ${LOGFILE} 2>&1
     popd > /dev/null 2>&1
 
+    # CSVs without a spec.replaces field are valid, so fall back to those if
+    # latest released version is unknown.
+    # Placeholder value is validated by operator-sdk during local bundle
+    # generation and so needs to conform to RFC1123.
+    if [[ -n "$BUNDLE_LATEST_RELEASED_VERSION" ]]; then
+        REPLACE_REGEX="$REPLACE_REGEX;s#---bundle-latest-released-version#${BUNDLE_LATEST_RELEASED_VERSION}#g"
```

Comment:
```suggestion
        REPLACE_REGEX="$REPLACE_REGEX;s#<<BUNDLE_LATEST_RELEASED_VERSION>>#${BUNDLE_LATEST_RELEASED_VERSION}#g"
```

Reply:
That was the first thing I tried, but operator-sdk actually validates the value in the `spec.replaces` field to ensure it conforms to RFC1123 ([or to an incorrect understanding of it](https://github.com/kubernetes/kubernetes/issues/94088), for the sdk version we use at this stage).

`<>_`  are not valid characters according to that so bundle generation fails. Sadly this forces us to break the convention we were following for replaceable values.

This is what I meant by the comment right above this line:
```
    # Placeholder value is validated by operator-sdk during local bundle
    # generation and so needs to conform to RFC1123.
```